Readme.md

```diff
@@ -31,7 +31,8 @@ $ touch tests/elm-verify-examples.json
   "root": "../src",
   "tests": [
     "Mock",
-    "Mock.Foo.Bar.Moo"
+    "Mock.Foo.Bar.Moo",
+    "./README.md"
```

Comment:
💭 I wonder if this should be relative from `"root"` above. It's unclear to me where this is relative to.
We probably also want to support `../README.md`.
@juanedi what do you think?

Reply:
also it fails with `content` isn't defined on `data`. If it doesn't find a file.

Reply:
nope, reconsidered. this is okay.